api/apiv3/api_event_controller.py

```diff
@@ -17,6 +18,33 @@
 from models.match import Match
 
 
+class ApiEventListAllController(ApiBaseController):
+    CACHE_VERSION = 0
+    CACHE_HEADER_LENGTH = 61
+
+    def _track_call(self, model_type=None):
+        action = 'event/list'
+        if model_type:
+            action += '/{}'.format(model_type)
+        self._track_call_defer(action, 'all')
+
+    def _render(self, model_type=None):
+        futures = []
+        for year in xrange(1992, tba_config.MAX_YEAR + 1):
```

Comment:
Do we have a lot of references to 1992 in the app? I know it doesn't change - but it might be worth setting as a `MIN_YEAR` in our config.

Reply:
Yeah we definitely do. We should start moving to `MIN_YEAR` in config, starting from this PR.

Reply:
:+1: I can file a cleanup ticket - seems fairly straightforward